core/migrations/0001_initial.py

```diff
@@ -58,23 +58,23 @@ class Migration(migrations.Migration):
                 ('platform', models.CharField(blank=True, help_text='Platform eg:maven', max_length=50)),
                 ('name', models.CharField(blank=True, help_text='Package reference name eg:org.apache.commons.io', max_length=50)),
                 ('version', models.CharField(blank=True, help_text='Reference version', max_length=50)),
-                ('package', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='vulnerabilities.Package')),
+                ('package', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='core.Package')),
```

Comment:
Sam comment as for `api`: core is too generic IMHO

Reply:
Should we keep the name `vulnerabilities` for the app containing the models instead? It's annoying to type but I can live with that. :)